src/sentry/api/endpoints/project_releases.py

```diff
@@ -58,6 +58,21 @@ def validate_version(self, attrs, source):
         return attrs
 
 
+def serialize_release_for_project(objects, user, project):
```

Comment:
it seemed like i'd have to do a decent amount of refactoring to be able to pass the additional project context necessary to make this work